controllers/front/CartController.php

```diff
@@ -581,8 +581,8 @@ private function areProductsAvailable()
         }
         if ($product['active']) {
             return $this->trans(
-                'The item %product% in your cart is no longer available in this quantity. You cannot proceed with your order until the quantity is adjusted.',
-                array('%product%' => $product['name']),
+                'The item %product% in your cart is no longer available in this quantity. You cannot proceed with your order until the quantity is adjusted. Maximum quantity is %quantity%',
```

Comment:
@Quetzacoalt91 I tried to disable product and the message "product is no longer available" is shown. So I believe it's working. Or maybe I misunderstood you?

Reply:
What I meant is the maximum quantity is not needed here. A customer cannot order the product because it was disabled, not because he reached the maximum quantity.